specification.md

```diff
@@ -4,73 +4,73 @@ Opportunity:
+* Data are becoming large enough that moving serialized objects around will soon be infeasible -- cloud-optimized formats will be required to support the next analysis phase, and out-of-core processing is becoming increasingly important.
 
-We envision an API that enables users to slice and compute on large (100s millions observations) single cell datasets stored in the cloud using the AnnData, Seurat, or Bioconductor toolchains.
+We envision an API that enables users to slice and compute on large (100s of millions of observations) single-cell datasets stored in the cloud using the AnnData, Seurat, or Bioconductor toolchains.
 
-To get started, we will focus on enabling data to be used by all major toolchains and on nailing the multi-modal data use cases. We believe this will be sufficient to drive API adoption. With data silos broken, larger data use cases will be enabled and the need for a cloud-optimized data format will be more widely felt.
+To get started, we will focus on enabling data to be used by all major toolchains and on nailing the multimodal-data use cases. We believe this will be sufficient to drive API adoption. With data silos broken, larger-data use cases will be enabled and the need for a cloud-optimized data format will be more widely felt.
 
-Initial Focus:
+Initial focus:
 
 * A standardized single-cell container, with (basic) read & query access to the data in the container.
-* Import/export from all commonly used in-memory formats (eg, AnnData, SingleCellExperiment, Seurat)
-* Access to underlying native (eg, TileDB) objects to allow advanced use cases
+* Import/export from all commonly used in-memory formats (e.g., AnnData, SingleCellExperiment, Seurat)
+* Access to underlying native (e.g., TileDB) objects to allow advanced use cases
 * Python and R support
 
-Longer term goals:
+Longer-term goals:
 
-* Native support in the popular toolchains, reducing the reliance on on-the-fly conversion (eg, [anndata2ri](https://github.com/theislab/anndata2ri))
+* Native support in the popular toolchains, reducing the reliance on on-the-fly conversion (e.g., [anndata2ri](https://github.com/theislab/anndata2ri))
 * Incremental modification and composition of the dataset
 * Optimizations required for performant analysis
 
 The core functions of the initial API are:
 
-* Compose a "sc dataset" Python object out of pre-existing TileDB arrays.  This object is the composition of one or more sc_groups (terms defined below).
-* Simple access to sc_dataset/sc_group properties (eg, obs, var, X) and ability to slice/query on the entire object based on obs/var labels.
-* For Python, Import/export an in-memory AnnData object (for subsequent use with AnnData/ScanPy) from _either_ a slice/query result or the entire object.  For R, same basic function but to/from Seurat and SingleCellExperiment
+* Compose a _SOMA collection_ object out of pre-existing TileDB arrays.  This object is the composition of one or more _SOMAs_(terms defined below).
```

Comment:
The SOMA acronym is introduced with no context and not introduced until the Data Model section.